src/components/common/SendCode/layouts/CountLabel.vue

```diff
@@ -0,0 +1,26 @@
+<template>
```

Comment:
Why we can't define it in counter component?

Reply:
it was in first [commit](https://github.com/endpass/endpass-auth/pull/280/commits/ebde2edbc93c7728326b7d2029522bc3763cab19#diff-5dfee36818ad96de5d03b7d25efae559R3)
but for new structure prefer separate different views in components and try not to use `v-if / v-else` expressions in templates